src/components/ProhibitionIcon/ProhibitionIcon.js

```diff
@@ -2,29 +2,47 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 
+import { Tooltip } from '@folio/stripes-components';
+
 import css from './ProhibitionIcon.css';
 
 /**
  * Component for rendering the circle backslash character with given message as an aria-label
  *
  * @param {string} [ariaLabel] - A label or a translation id for the aria-label attribute
+ * @param {string} fieldName - field name for creation of required tooltip id attribute
  * @returns {{}}
  */
-export const ProhibitionIcon = ({ ariaLabel }) => {
+export const ProhibitionIcon = ({
+  ariaLabel, fieldName,
+}) => {
   const ariaLabelIsTranslationId = ariaLabel && ariaLabel.includes('ui-');
   const ariaLabelIsMessageString = ariaLabel && !ariaLabel.includes('ui-');
-  const translationId = ariaLabelIsTranslationId ? ariaLabel : 'ui-data-import.noValueAllowed';
```

Comment:
Please don't get rid of this translation - screen readers should read the current icon as "No value allowed" (https://issues.folio.org/browse/UIDATIMP-471?focusedCommentId=74991&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-74991)
I'd suggest creating two separate messages - for `aria-label` and `Tooltip text`

Reply:
According to A-M comment we should update text